exercises/diffie-hellman/canonical-data.json

```diff
@@ -0,0 +1,126 @@
+{ "exercise": "diffie-hellman"
+, "version" : "1.0.0"
+, "comments":
+    [ "                                                        "
+    , " There are irregularities in how this problem is being  "
+    , " tested across the current tracks:                      "
+    , "                                                        "
+    , " csharp, fsharp, elixer and go all consistently test    "
+    , " both for large and small prime numbers test cases.     "
+    , " They also assume that the application generates the    "
+    , " private key, not the user picks as specified by the    "
+    , " problem description.                                   "
+    , "                                                        "
+    , " ecmascript and javascript, on the other hand, only use "
+    , " the very small prime numbers as the test cases, and    "
+    , " ask the user to implement a private key generator.     "
+    , "                                                        "
+    , " The other major difference is that both javascript and "
+    , " ecmascript test boundaries and verify that their       "
+    , " implementations throw errors for specific cases.       "
+    , " The other implementations do no do this (perhaps       "
+    , " because they are all typed?).                          "
+    , "                                                        "
+    , " One last difference is that the go implementation uses "
+    , " hexadecimal notation for the bit Ints."
```

Comment:
These comments are probably better put in the git commit message and/or the Github pull request description.

Reply:
Got it (though it's not clear from the template that these types are not what goes here).